shu-etal.ipynb

```diff
@@ -0,0 +1,1516 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "525fe7fa",
+   "metadata": {},
+   "source": [
+    "# Replication: Shu *et al*, 2020\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook attempts to replicate the following paper with the [PPMI](http://ppmi-info.org) dataset:\n",
+    "\n",
+    "<div class=\"alert alert-block alert-success\">\n",
+    "Shu, Zhen‐Yu, et al. <a href=\"https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.28522?casa_token=Ab53WvMlODcAAAAA%3AXcgDLmq8egqW7uwd2g3jY9jIljhLu3VhIbvMWgbcfoWOxjO_9H7Arf91t2FBZDZ8E94Je4Wmrn0ZmkeZ\">Predicting the progression of Parkinson's disease using conventional MRI and machine learning: An application of radiomic biomarkers in whole‐brain white matter.</a> Magnetic Resonance in Medicine 85.3 (2021): 1611-1624.</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "078e819a",
+   "metadata": {},
+   "source": [
+    "This study recruited 72 patients with progressive PD and 72 patients with stable PD matched by age, sex and category of HYS. This following population from PPMI was used:\n",
```

Comment:
notebook intro should explain briefly what the replicated paper did and what was the main result